app/src/main/java/ch/epfl/sweng/SDP/game/VotingPageActivity.java

```diff
@@ -39,14 +49,73 @@
 
     private String[] playersNames;
 
+    private ProgressBar progressBar;
     private ImageView drawingView;
     private TextView playerNameView;
+
     private RatingBar ratingBar;
 
     public int[] getRatings() {
         return ratings.clone();
     }
 
+    private final ValueEventListener listenerCounter = new ValueEventListener() {
```

Comment:
This is to interact with Firebase database right?
Remember that Andrea wrote a helper class to update and get stuff from database.

Reply:
The helper class is Database.java, and for the moment it only has a better getReference() method

Reply:
Yes you are! Don't worry ;)